tests/python/pants_test/util/test_objects.py

```diff
@@ -41,17 +39,24 @@ def test_none(self):
     with self.assertRaises(ValueError):
       SubclassesOf()
 
+  def test_str_and_repr(self):
+    superclasses_of_b = SuperclassesOf(self.B)
+    self.assertEqual("-B", str(superclasses_of_b))
+    self.assertEqual("SuperclassesOf(B)", repr(superclasses_of_b))
+
+    superclasses_of_multiple = SuperclassesOf(self.A, self.B)
+    self.assertEqual("-(A, B)", str(superclasses_of_multiple))
+    self.assertEqual("SuperclassesOf(A, B)", repr(superclasses_of_multiple))
```

Comment:
let's use ` or ` rather than `, ` to join these; it's non-obvious what `,` means

Reply:
I completely agree!